glob_test.go

```diff
@@ -90,6 +90,7 @@ func TestGlob(t *testing.T) {
 		"* ",    // Trailing white space
 		" *",    // Leading white space
 		"*ʤ*",   // Non-matching unicode
+		"this*this is a test",  // Repeated prefix
```

Comment:
This is not valid `gofmt`-compatible formatting. You should use `gofmt` to format it.